examples/nlp/language_modeling/tuning/conf/megatron_gpt_peft_eval_config.yaml

```diff
@@ -105,16 +105,17 @@ model:
       drop_last: False
       context_key: ${data.train_ds.context_key}
       label_key: ${data.train_ds.label_key}
-      add_eos: ${data.train_ds.add_eos}
+      add_eos: False
       add_sep: ${data.train_ds.add_sep}
       add_bos: ${data.train_ds.add_bos}
       separate_prompt_and_response_with_newline: ${data.train_ds.separate_prompt_and_response_with_newline}
       write_predictions_to_file: False
       output_file_path_prefix: null # Prefix of the file to write predictions to.
-      truncation_field: ${data.train_ds.truncation_field} # Options: ['context', 'answer']
+      truncation_field: ${data.train_ds.truncation_field} # Options: keys in context_key
       index_mapping_dir: null # Path to a directory to write index mapping files.
       prompt_template: ${data.train_ds.prompt_template}
       tokens_to_generate: 32 # decide how many tokens we want to generate to evaluate performance with string metrics
+      truncation_augmentation: False # Do truncation from end
```

Comment:
Just eyeballing this variable name and it's comment, its not super obvious what exactly it does

Reply:
Change to `truncation_method` with options `['random', 'left', 'right']`